.gitattributes

```diff
@@ -1 +1,6 @@
 web/vtadmin/src/proto/** linguist-generated=true
+web/vtadmin/src/proto/*.js -diff
+web/vtadmin/src/proto/*.ts -diff
+go/vt/proto/** linguist-generated=true
+go/vt/proto/*/*.pb.go -diff
```

Comment:
I like it! Any reason not to use `-merge` for the generated files too? Then the merge will be treated as if they were binary files too and it will just use the entire latest file. I don't think I've seen any merge conflicts with the generated files though... so not likely needed. ❤️

Reply:
Don't get the use case you are referring to: we regenerate these files in its entirety and any merge conflicts would need to be fixed in the .proto file.

Will merge this PR as-is and we can extend this for your use-case separately.